serverless-chatbot/README.md

```diff
@@ -0,0 +1,15 @@
+# Serverless Chatbot
+
+This is an example project that shows how to create a serverless chatbot that interacts with Beekeeper users and is triggered by `CHATS.MESSAGE.CREATED` webhook event.
+
+## Prerequisites 
+1. Make sure you have Node.js installed on your machine.
+2. Follow [the instructions](https://serverless.com/framework/docs/providers/aws/guide/installation/) to install the Serverless framework.
+3. Install [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) and [configure the credentials](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html).
+
```

Comment:
We should consider adding instructions for setting up a bot and a webhook (or at least point to the documentation?).

Reply:
Yeah, I was going to do that. I also added a script for making the request.